common/environment/build-style/meson.sh

```diff
@@ -1 +1,9 @@
 hostmakedepends+=" meson"
+
+export PKG_CONFIG_ALLOW_CROSS=1
+
+export LIBGIT2_SYS_USE_PKG_CONFIG=1
+export GETTEXT_BIN_DIR=/usr/bin
+export GETTEXT_LIB_DIR="${XBPS_CROSS_BASE}/usr/lib/gettext"
+export GETTEXT_INCLUDE_DIR="${XBPS_CROSS_BASE}/usr/include"
+export LIBSSH2_SYS_USE_PKG_CONFIG=1
```

Comment:
missing newline here

Reply:
I would prefer a solution which doesn't involve maintaining two copies of this list

Reply:
Hm, can we share envs between build styles? Adding it to the global env seems meh to me

Reply:
@jnbr #7372